src/certs/snp/ca/mod.rs

```diff
@@ -29,6 +31,77 @@ impl<'a> Verifiable for &'a Chain {
     }
 }
 
+#[cfg(feature = "openssl")]
+impl From<(X509, X509)> for Chain {
```

Comment:
I think the ordering of the tuples/arrays/etc should be ARK/ASK rather than ASK/ARK.

Reply:
I don't disagree that I would like the root key first, however we need to follow the standard already presented by the AMD Key Distribution Server (KDS) which explicitly returns (ASK, ARK) in PEM format:

https://www.amd.com/content/dam/amd/en/documents/epyc-technical-docs/specifications/57230.pdf#page=17